packages/remix-edge-adapter/package.json

```diff
@@ -36,7 +36,8 @@
   },
   "homepage": "https://github.com/netlify/remix-compute#readme",
   "dependencies": {
-    "@netlify/remix-runtime": "*"
+    "@netlify/remix-runtime": "*",
```

Comment:
I know this is not introduced in this PR - but from what I can tell this wasn't published yet at least - we really shouldn't have `*` in dependencies (devDependencies still not great, but not as problematic) - if you do this and we ever do a breaking change in `@netlify/remix-runtime` people on older versions will have troubles if they ever nuke their lock file as package managers will just install latest one that might or might not be compatible

Reply:
The remix-edge-adapter has already been released. I thought in a monorepo context, we were supposed to use `*` for deps are another package in the monorepo?

Reply:
https://www.runpkg.com/?@netlify/remix-edge-adapter@1.2.0/package.json#39 shows concrete version on latest recent version, so that's why I mentioned that `*` being in deps was not released to npm yet.

> I thought in a monorepo context, we were supposed to use * for deps that are another package in the monorepo?

This is "I don't want to deal with it solution" to problems of installing package from npm instead of using version of the package in monorepo. 

For monorepo it makes sense because `*` will always allow to use version of package that is in monorepo, but if you publish the package to npm - that `*` is also used by users and not only by devs in monorepo itself. And while "right now" it likely won't cause problems, sometime in to the future it can if there is major version bump that is not compatible with previous major as users on older `@netlify/remix-edge-adapter` could install newer `@netlify/remix-runtime` (as all version constraints would still be satisfied with `*`) that might not actually be compatible.

Reply:
Just as example `@types/*` packages tend to use `*` version selector for other `@types/*` deps and it often cause problems for their consumers and make consumer struggling to resolve those issues that they shouldn't really have to deal with if version selectors were explicit and not wildcards

Reply:
I think this is something release-please can handle